<ui version="4.0" >
 <class>NewContactListWindowClass</class>
 <widget class="QWidget" name="NewContactListWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>388</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>185</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="styledContactsFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>18</number>
      </property>
      <property name="bottomMargin" >
       <number>6</number>
      </property>
      <item>
       <widget class="NewContactListGripWidget" native="1" name="gripWidget" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>6</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>6</width>
          <height>10000</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" >
        <item>
         <widget class="QFrame" name="buttonsSetupFrame" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth" >
           <number>0</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>6</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>6</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" >
             <property name="spacing" >
              <number>0</number>
             </property>
             <property name="sizeConstraint" >
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QToolButton" name="findButton" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="checkable" >
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle" >
                <enum>Qt::ToolButtonTextOnly</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="verticalLineLabel" >
               <property name="minimumSize" >
                <size>
                 <width>1</width>
                 <height>0</height>
                </size>
               </property>
               <property name="lineWidth" >
                <number>0</number>
               </property>
               <property name="pixmap" >
                <pixmap resource="moodbox.qrc" >:/MoodBox/Resources/button_vertical_line.png</pixmap>
               </property>
               <property name="scaledContents" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="setupToolButton" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="checkable" >
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle" >
                <enum>Qt::ToolButtonTextOnly</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="contactListFrame" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth" >
           <number>0</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="MoodBox::NewContactListWidget2" name="newContactListWidget" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="acceptDrops" >
              <bool>true</bool>
             </property>
             <property name="frameShape" >
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Plain</enum>
             </property>
             <property name="horizontalScrollBarPolicy" >
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="editTriggers" >
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="dragDropMode" >
              <enum>QAbstractItemView::DropOnly</enum>
             </property>
             <property name="iconSize" >
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
             <property name="resizeMode" >
              <enum>QListView::Adjust</enum>
             </property>
             <property name="layoutMode" >
              <enum>QListView::Batched</enum>
             </property>
             <property name="uniformItemSizes" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <customwidgets>
  <customwidget>
   <class>NewContactListGripWidget</class>
   <extends>QWidget</extends>
   <header>newcontactlistgripwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MoodBox::NewContactListWidget2</class>
   <extends>QListView</extends>
   <header>newcontactlistwidget2.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="moodbox.qrc" />
 </resources>
 <connections/>
</ui>
